daml-script/export/src/main/scala/com/daml/script/export/Config.scala

```diff
@@ -53,8 +53,11 @@ object Config {
     opt[String]("party")
       .required()
       .unbounded()
-      .action((x, c) => c.copy(parties = x :: c.parties))
-      .text("Export ledger state as seen by these parties.")
+      .action((x, c) => c.copy(parties = x.split(",").toList ++ c.parties))
```

Comment:
afaik you don’t have to split this yourself, you can specify a `Seq[String]` and scopt will do it for you. see https://github.com/digital-asset/daml/blob/2935b6fe84c5e9abd10e25849c838577249f5e77/ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Cli.scala#L146 for example

Reply:
Nice, thanks for the pointer!